### PR TITLE
Feature/40385 [TextField]

### PR DESCRIPTION
**Pull request by suraj-aggarwal**
[issue no. 40385](https://redmine.successive.tech/issues/40385)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "ecmaFeatures": {
+        "jsx": true,
+        "classes": true
+      }
+    }
+  }
+
```
Comment (madhavBansal-successiveTech) [inline .eslintrc]: Fix lint

File: src/components/TextField/TextField.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, style } from './style';
+
+function TextField(props) {
+  const { value, disabled, error } = props;
+  return (
+    <div>
+      <Input type="text" value={value} disabled={disabled} />
+      <p style={style}>{error}</p>
```
Comment (madhavBansal-successiveTech) [inline src/components/TextField/TextField.jsx]: Only use error p tag if error is come

File: src/components/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export { TextField } from './TextField/index';
```
Comment (madhavBansal-successiveTech) [inline src/components/index.js]: Export without the use of index

File: src/pages/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export { TextFieldDemo as default } from './TextFieldDemo/index';
```
Comment (madhavBansal-successiveTech) [inline src/pages/index.js]: use export default as

File: .eslintrc (hunk 1 above)
Comment (suraj-aggarwal) [reply .eslintrc]: Done

File: src/components/TextField/TextField.jsx (hunk 2 above)
Comment (suraj-aggarwal) [reply src/components/TextField/TextField.jsx]: Dine

Comment (suraj-aggarwal) [reply src/components/TextField/TextField.jsx]: Done

File: src/components/index.js (hunk 3 above)
Comment (suraj-aggarwal) [reply src/components/index.js]: Done

File: src/pages/index.js (hunk 4 above)
Comment (suraj-aggarwal) [reply src/pages/index.js]: Done

**Pull request merged**